force-app/main/default/classes/GiftService.cls

```diff
@@ -42,4 +42,9 @@ public with sharing class GiftService {
         GiftDTO giftDTO = new GiftDTO(gift);
         return new GiftView(giftDTO);
     }
+
+    public void save(Gift gift) {
```

Comment:
It looks like this method only applies to SGe but the word "save" feels like it could apply to both SGE and BGE (to me at least). Would it make sense to change the name to be more SGE-specific? Thinking something like `saveAndProcess()` or even `saveSingleGift()` to mirror the calling function?

Reply:
Updated!